### PR TITLE
Create add-Ip-to-existing-paid-plan.md

### PR DESCRIPTION
**Pull request by mitchwh24**
Another FAQ to offset customer queries

### CODE REVIEW

File: articles/faq/add-Ip-to-existing-paid-plan.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: "Can I add a Dedicated IP to my existing Paid plan?"
```
Comment (ewandennis) [inline articles/faq/add-Ip-to-existing-paid-plan.md]: Do we need the capitalised D and P?

File: articles/faq/add-Ip-to-existing-paid-plan.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+description: "The older paid plans no longer offer the ability to add dedicated IPs to your account. If you require Dedicated IPs please upgrade to one of our new plans from the Accounts/Billing page within the SparkPost UI (https://app.sparkpost.com/account/billing).
```
Comment (ewandennis) [inline articles/faq/add-Ip-to-existing-paid-plan.md]: Lowercase D in `dedicated`? Maybe remove the URL from the description.

File: articles/faq/add-Ip-to-existing-paid-plan.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+Please note that new plans from 150K upwards include a dedicated IP, allowing you to add an IP Pool and new IP. You can of course add multiple IPs at 20USD per month in that or other IP Pools you create."
```
Comment (ewandennis) [inline articles/faq/add-Ip-to-existing-paid-plan.md]: This 2nd line in the description may not work properly. Maybe just remove it and leave the detail in the body below?

File: articles/faq/add-Ip-to-existing-paid-plan.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+---
+
+The older paid plans no longer offer the ability to add dedicated IPs to your account. If you require Dedicated IPs please upgrade to one of our new plans from the Accounts/Billing page within the SparkPost UI (https://app.sparkpost.com/account/billing).
```
Comment (ewandennis) [inline articles/faq/add-Ip-to-existing-paid-plan.md]: Remove `...within the SparkPost UI` and make Accounts/Billing page a link like this:

`[Accounts/Billing page](https://app.sparkpost.com/account/billing)`

File: articles/faq/add-Ip-to-existing-paid-plan.md (hunk 1 above)
Comment (mitchwh24) [inline articles/faq/add-Ip-to-existing-paid-plan.md]: I tend to think of a Ded IP as a thing - so tend to capitalise - P in paid i can drop

**Pull request merged**